colorpair_num_test.py

```diff
@@ -0,0 +1,21 @@
+import color_pairnum_translator
+
+
+def test_number_to_pair(pair_number,

```

Comment:
Nice separation of tests. See if you can think of test-names that include their expectations. Do additional test-possibilities emerge?